src/renderer/account/components/SendPanel/index.js

```diff
@@ -31,6 +36,11 @@ export default compose(
 
   withData(authActions, mapAuthDataToProps),
   withNetworkData(),
+  withInitialCall(balancesActions, ({ net, address }) => ({ net, address })),
```

Comment:
There's a stealthy issue here, and it's that both `SendPanel` and `AccountPanel` are being rendered at the same time, and both of them include `withInitialCall(balancesActions, ...)`.  As a result, we'll end up fetching balance data twice, which is already an expensive call.  Now that we have two child components that need this data, this call should be moved up one level into `account/components/index.js`.

Reply:
I would also suggest moving the withProgressComponents HOC call up a level from account/components/AccountPanel/index.js into account/components/index.js. That way, the account screen waits for all balance data to load before trying to render.

In conjunction with that, I believe we can remove the `withLoadingProp` and `withProgressChange` HOC calls further down in this file.